code/drasil-example/Drasil/Projectile/Unitals.hs

```diff
@@ -1,16 +1,19 @@
 module Drasil.Projectile.Unitals where
 
 import Language.Drasil
-import Language.Drasil.ShortHands
+import Language.Drasil.ShortHands (lD, lTheta, lV, lP, lT, lS, vEpsilon)
 
 import Data.Drasil.Quantities.Math (pi_)
 
 import Data.Drasil.Constraints (gtZeroConstr)
 import Data.Drasil.SI_Units (radian, metre, second)
 import Data.Drasil.Units.Physics (velU)
 
-import qualified Drasil.Projectile.Concepts as C (flightDur, offset,
-  flightDur, landPos, launAngle, launSpeed, offset, targPos)
+import qualified Drasil.Projectile.Concepts as C
+
+velVecSpeed, speed1DAcc :: UnitalChunk
+velVecSpeed = uc C.velVecSpeed (sup (vec lV) $ Label "speed") velU
+speed1DAcc  = uc C.speed1DAcc  (sup lV $ Label "f") velU
```

Comment:
The f gets attached here - but why? I'm no longer sure what this quantity even refers to! (See previous comments).